docs/modules/operation/pages/deep-dive/admin/webui/grafana-dashboard-box.adoc

```diff
@@ -22,78 +24,81 @@ image::webui/startpage/01_grafana-box.png[]
 | Default
 
 | org.opennms.grafanaBox.show
-| This setting controls whether a grafana box showing the available dashboards is placed on the landing page.
+| Determines whether a Grafana Dashboard Box showing the available dashboards is included on the home page.
 | Boolean
-| false
+| `false`
 
 | org.opennms.grafanaBox.hostname
 | The hostname of the Grafana server.
 You may need to use the IP address or fully qualified domain name, based on your DNS records.
 | String
-| localhost
+| `localhost`
 
 | org.opennms.grafanaBox.port
-| The port of the Grafana server REST API.
+| The port for the Grafana server's REST API.
 | Integer
-| 3000
+| `3000`
```

Comment:
These look like back-ticks and not single quotes.  Not sure what we need or want to use but calling it out.